bin/fetch-font.sh

```diff
@@ -1,3 +1,3 @@
-#!/bin/bash
+#!/bin/sh -e
 
 wget -O public/fonts/Hack-Regular.ttf https://github.com/source-foundry/Hack/raw/master/build/ttf/Hack-Regular.ttf
```

Comment:
I have a preference of doing this kind of thing as a Makefile. (This is how you make this file.) But w/e.

Reply:
Yeah, that's not something I wrote in this PR, and I'm explicitly leaving all the packaging and buildsystem work for a future PR.